### ISSUE TITLE
cdn hardcoded paths is not ideal

### ISSUE THREAD

**Issue by steverweber** [django-tiptap/django_tiptap#19]
cdn hardcoded paths is not ideal. Perhaps make it a variable so we can work-towards using a the site's static path.

**Comment by sereneinserenade** [comment]
Yes, this should be the ideal case, will discuss the specs with @solen0id
